### ISSUE TITLE
Collapse animation speed

### ISSUE THREAD

**Issue by ValiDraganescu** [proksh/atomize#32]
Hi, I am using atomize for a toy project. I noticed that the collapse atom is not very snappy in it's animation, I'd like to speed up the animation. How to do this? Thank you!

**Comment by ValiDraganescu** [comment]
In Collapse.js I see 
```
render() {
    const { defaultHeight } = this.state;
    const { p, children, isOpen, ...rest } = this.props;

    /**
     * Duration for the transition
     */
    const duration = 400;
...
```
This means the duration is hardcoded?

**Comment by proksh** [comment]
Yes @ValiDraganescu, It is hardcoded right now. I will soon update it in the next version, for custom transitions.

**Comment by ValiDraganescu** [comment]
I ended up copying Collapse.js and modifying per my needs, it works for now

**Comment by proksh** [comment]
Great @ValiDraganescu

**Comment by bobbylemm** [comment]
@proksh I would like to know if this issue has been fixed

**Comment by LucaMoro96** [comment]
@bobbylemm Not yet!

**Comment by bobbylemm** [comment]
@LucaMoro96 can I work on this

**Comment by LucaMoro96** [comment]
@bobbylemm Sure!

**Comment by bobbylemm** [comment]
Alright great

**Comment by manan1979** [comment]
hi @ValiDraganescu can i work on this?

**Comment by ValiDraganescu** [comment]
Hi @manan1979 , this is not my repo, I see others above offering to work on this one, so maybe check with @bobbylemm
